tests/cli/commands/test_fastapi_api_command.py

```diff
@@ -168,3 +169,22 @@ def test_cli_fastapi_api_args(self):
                 ],
                 close_fds=True,
             )
+
+    @pytest.mark.parametrize(
+        "ssl_arguments",
+        [["--ssl-cert", "_.crt", "--ssl-key", "_.key"], ["--ssl-cert", "_.crt"], ["--ssl-key", "_.key"]],
+    )
+    def test_get_ssl_cert_and_key_filepaths_with_incorrect_usage(self, ssl_arguments):
+        args = self.parser.parse_args(["fastapi-api"] + ssl_arguments)
+        with pytest.raises(AirflowConfigException):
```

Comment:
All if the raised exceptions are `AirflowConfigException`, I'd add a `match=...` to ensure that each fails for the expected error.

Reply:
Ah yep, have updated to use `match=...`.